pkgdb2client/__init__.py

```diff
@@ -41,6 +41,9 @@ def emit(self, record):
 
 __version__ = pkg_resources.get_distribution('packagedb-cli').version
 PKGDB_URL = r'https://admin.fedoraproject.org/pkgdb/'
+FAS_URL = r'https://admin.fedoraproject.org/accounts'
+BZ_URL = r'https://bugzilla.redhat.com/xmlrpc.cgi'
+KOJI_HUB = r'http://koji.fedoraproject.org/kojihub'
```

Comment:
I like the idea but that it's a little outside the scope of this PR as I would like to cut a release somewhat soon, we already have piled some interesting changes :)